components/contracts/hitsCategoryService.ts

```diff
@@ -426,8 +443,14 @@ export class HitsCategoryService {
             const paginated = paginate(hitObjectList, this.hitsPerPage)
 
             hitsCategory.Data.Hits = paginated[pageNumber]
-            hitsCategory.Data.HasMore = paginated.length > pageNumber + 1
-            hitsCategory.CurrentSubType = `${category}_${filter}`
+            hitsCategory.Data.HasMore =
+                categoryName === "Featured"
+                    ? true
```

Comment:
What if we don't have more?

Reply:
When we don't have more, the next page will display official featured contracts instead.

Reply:
What if we're out of official featured contracts?

Reply:
When official ones are fetched instead, it won't reach this line. Check out the newly added `fetchFromOfficial` block above.